apps/webpack.js

```diff
@@ -39,15 +41,12 @@ var baseConfig = {
       http: 'stream-http',
     }
   },
-  sassLoader: {
-    includePaths: [path.resolve(__dirname, '..', 'shared', 'css')]
-  },
   module: {
     loaders: [
       {test: /\.json$/, loader: 'json-loader'},
       {test: /\.ejs$/, loader: 'ejs-compiled-loader'},
       {test: /\.css$/, loader: 'style-loader!css-loader'},
-      {test: /\.scss$/, loader: 'style-loader!css-loader!sass-loader'},
+      {test: /\.scss$/, loader: `style-loader!css-loader!sass-loader?includePaths=${scssIncludePath}`},
```

Comment:
This looks like we'll _only_ convert scss that's in /shared/css, not anywhere else.  Is that the desired behavior, or am I not reading this correctly?

Reply:
I think it converts .scss files everywhere, but the `includePaths` hint tells `node-sass` where to look for resolving `@import` statements: https://github.com/sass/node-sass#includepaths.

This shouldn't change the behavior at all — just moves the option from top-level `sassLoader` config option to the loader rule.

Reply:
Oh, of course! :man_facepalming: I should know that.  LGTM!

Reply:
No worries, I didn't realize it was an option passed to `node-sass` and not webpack!